imagery-cesium/maps/cesium-map/cesium-map.ts

```diff
@@ -488,7 +488,12 @@ export class CesiumMap extends BaseImageryMap<any> {
 	}
 
 	addGeojsonLayer(data: GeoJsonObject) {
-		throw new Error('Method not implemented.');
+		Cesium.GeoJsonDataSource.load(data).then(lyr => {
```

Comment:
what kind of layer is this?
should it be created as cesiumLayer?
how can we remove or get this layer?